### ISSUE TITLE
Adapt links in emails to user's language

### ISSUE THREAD

**Issue by Powersource** [dukop/django-dukop#427]
Right now all links are danish which makes the page (esp admin) harder to read.

**Comment by benjaoming** [comment]
Yes, agreed - emails are kept in the same locale as the event session that spawned their creation.
